packages/tuff-intelligence/src/tools/types.ts

```
@@ -0,0 +1,103 @@
+import type { z } from 'zod'
+
+export type TuffToolRiskLevel = 'low' | 'medium' | 'high' | 'critical'
+export type TuffToolSource = 'builtin' | 'plugin' | 'mcp' | 'skill' | 'workflow'
+
+export type TuffToolErrorCode =
```

ESLint

[error] '=' should be placed at the beginning of the line.

(style/operator-linebreak)